storage/innobase/log/log0log.cc

```diff
@@ -382,7 +382,8 @@ class file_pmem_io final : public file_io
                                                                    : DB_ERROR;
   }
   dberr_t close() noexcept final { return m_file.unmap(); }
-  dberr_t read(os_offset_t offset, span<byte> buf) noexcept final
+  dberr_t MY_ATTRIBUTE((warn_unused_result)) read(os_offset_t offset,
+                                                  span<byte> buf) noexcept final
```

Comment:
Please add the attribute on a separate line, right before the function return type `dberr_t`. It’s nicer for `grep` if the main function signature fits in a single line.